src/main/java/com/napier/sem/City.java

```diff
@@ -11,11 +11,11 @@
  */
 public class City
 {
-    // Private properties
-    private String name;
-    private String country;
-    private String district;
-    private Integer population;
+    // Private properties - Removed private so tests would work, only work around we could think of for now.
```

Comment:
this is fine for now, i understand the problem. we'll add getters to our classes at some point.

Reply:
I was hoping that we would be able to do this without getters and setters

Reply:
i know, they're ugly